Danki2/Assets/Scripts/Actor/Passive/Passive.cs

```diff
@@ -2,6 +2,6 @@
 using System.Collections.Generic;
 using UnityEngine;
 
-public class Passive : Actor
+public abstract class Passive : Actor
```

Comment:
This class will have to implement `Act()` but it can just be an empty method with a TODO for now I think